app/models/task.rb

```diff
@@ -1,3 +1,19 @@
 class Task < ActiveRecord::Base
+  belongs_to :person
 
+  def change_complete_status(complete_request)
+    complete_request == "1" ? complete : uncomplete
+  end
+
+  def complete
+    update(completed_at: Time.now)
+  end
+
+  def uncomplete
+    update(completed_at: nil)
+  end
+
+  def owner
+    Person.find(self.person_id).name
+  end
```

Comment:
I assume that this is another case where `person.name` would be sufficient for the body of the method :) I always struggle with naming my methods. I just wanted a quick method call that returns the "name" of the person that the task belongs to.... presumably `#person_name` would've been more appropriate?

Should I just call `task.person.name` every time, or is this method actually useful to have?